wpeverest/everest-forms
# Fix - Payment Detail left right spacing

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes payment detail spacing issue.
![image](https://user-images.githubusercontent.com/66351929/108946942-935de980-7687-11eb-8b00-ee80aab3f5f2.png)

### How to test the changes in this Pull Request:
1. Checkout to this branch
2. Fill the form and checkout the entries page design issue
3. Please check if the design issue still exist or not

### Types of changes:




### Other information:




### Changelog entry

> Fix - Payment Detail left right spacing
